### PR TITLE
Bug 78/v4

### PR DESCRIPTION
**Pull request by inashivb**

### CODE REVIEW

File: tests/bug-78-http-uri/test.yaml [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+ - -k none
+
+checks:
+- filter:
+    count: 1
+    match:
+      alert:
+        action: allowed
+        category: ''
+        gid: 1
```
Comment (catenacyber) [inline tests/bug-78-http-uri/test.yaml:15]: Maybe some fields are not important... cf creates...

**Comment by catenacyber** [comment]
Looks overall good to me, event if it can be improved by removing some parts of the checks (like undefined gid of the signature where default value gets used)

**Comment by inashivb** [comment]
I was just redoing this and found out that suricata-verify would fail in case any of the inner levels of yaml tree do not have **all** the fields available. I don't know if it was always like this or was introduced recently. I'll check it out. Created ticket: https://redmine.openinfosecfoundation.org/issues/4276

**Comment by inashivb** [comment]
Replaced with #429

**Pull request closed without merging**